operator/cmd/mesh/manifest-apply.go

```diff
@@ -73,6 +82,10 @@ func manifestApplyCmd(rootArgs *rootArgs, maArgs *manifestApplyArgs) *cobra.Comm
   # Enable grafana dashboard
   istioctl manifest apply --set values.grafana.enabled=true
 
+ # plugin in existing CA Certificates from cert-dir /samples/certs
```

Comment:
Do we want to say "if the directory doesn't include certificates, we will generate a root and intermediate certificate" ? 

BTW - if we really want to do it right, we should ask for a param including the cluster name, so we can include it in the cert.

Reply:
added an example for `if the directory doesn't include certificates, we will generate a root and intermediate certificate"

Reply:
Can we take the cluster name from the profile? We already have global.multiCluster.clusterName which is required for multi-cluster. Maybe we make that a top-level field and require that its always set (or pick a random default / UID).

Reply:
+1. After a quick search I see at least a couple of places where cluster name is specified:

"cluster" in https://github.com/istio/istio/blob/master/istioctl/cmd/proxyconfig.go#L627
"name" in https://github.com/istio/istio/blob/master/istioctl/pkg/multicluster/remote_secret.go#L322

I think specifying a single top-level flag for this makes a lot of sense and it would eliminate the inconsistent naming of the property as well.

Reply:
The service registry name in mesh network configuration also needs to match cluster name.

We use cluster name and cluster ID inter changeably. We should pick one. ID would be better since we're assuming uniqueness. 

I disagree about using a top-level flag for setting cluster ID. We shouldn't mix declarative and imperative config. if the cluster ID is set in the profile, `istioctl manifest apply` can use it for generating the intermediate. If its not we have two options: (1) throw an error or (2) use/create a UUID for the default clusterName in the profile and the intermediate cert. One option for UUID is the UUID of the kube-system namespace in the target cluster. It is unique and fixed for the lifetime of the cluster. The default cluster ID generation would be idempotent.

Reply:
> I disagree about using a top-level flag for setting cluster ID

I'm a little confused, given:

> We already have global.multiCluster.clusterName which is required for multi-cluster. Maybe we make that a top-level field and require that its always set (or pick a random default / UID).

I thought I was agreeing with your point :)

Reply:
what is the need for cluster name here?

Reply:
It is used as the org name in the generate cert.

Reply:
added `clusterID` into global.yaml and if no `values.global.clusterID` provided will use the UUID from `kube-system`,

Reply:
> I thought I was agreeing with your point :)

Ah, I interpreted "top-level flag" to mean `istioctl manifest apply -f <profile.yaml --cluster_id=<....`. What @irisdingbj has in the latest set of changes is what I think we both had in mind.

Reply:
We can split this PR in 2 parts - one to generate the cert, the other to add clusterName and possibly replace it with clusterID (I don't see the diff and I'm sure clusterName was bikesheded enough) .